index.html

```diff
@@ -17,22 +17,22 @@
         <span class="logo">Joy</span>
       </button>
   
-        <ul class="ul-nav" title="navigation-list">
-         <li><a href="#portfolio">Portfolio</a></li>
-         <li><a href="#about">About</a></li>
-         <li><a href="#contactme">Contact</a></li>
-        </ul>
-        
-      <button class="nav-button2">
-        <span> 
-          <img src="./sources/Hamburg" alt="hamburg">
-        </span>
-      </button>
+      <ul class="ul-nav" title="navigation-list">
+        <li><a class="navigationList" href="#portfolio">Portfolio</a></li>
+        <li><a class="navigationList" href="#about">About</a></li>
+        <li><a class="navigationList" href="#contactme">Contact</a></li>
+      </ul>
+    
+      <div class="nav-button2">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
```

Comment:
- [Optional] On the desktop display none of your menus are visible. This is a big concern when dealing with a responsive webpage. Please address this as this will make your design incomplete.
![Screenshot 2023-01-05 093244](https://user-images.githubusercontent.com/69077061/210716514-0aee0d4e-9987-460d-ad40-2c500d6266c3.png)